package.json

```diff
@@ -16,21 +16,23 @@
     "jasmine": "^3.5.0",
     "jasmine-spec-reporter": "^5.0.2",
     "lerna": "^3.18.4",
+    "crypto": "^1.0.1",
     "ts-node": "^8.6.2",
     "typedoc": "^0.17.8",
     "typescript": "^3.9.7"
   },
   "scripts": {
     "build": "lerna run compile",
     "dev-build": "lerna run dev-compile --parallel --concurrency 10",
-    "postinstall": "lerna clean --yes && lerna bootstrap && node ./tools/init_dev.js -f",
+    "preinstall": "yarn workspace @pipcook/boa run install-dep",
```

Comment:
Could we move this to boa package?

Reply:
`yarn` will not trigger `postinstall` & `preinstall` in the internal packages if no additional packages are downloaded, therefore, we need to put the two scripts at the root `package.json`

Reply:
This doesn't work with the separate build :(